running the reference implementation one-click in Docker container

izinin opened the pull request:
Hello Philip

Please review my addition to the project that allows as2-peppol-server running in container. It generates self-signed certificate and setups correct paths for data and keystore. When container is running it can receive PEPPOL documents from third-party PEPPOL implementation. I think it is useful in testing.

Regards,
Igor

Commit:
running the reference implementation one-click in Docker container

phax commented:
Igor - Thanks a million!
That is really really helpful! Love it!

There is a niftly little detail I dislike and it is the change of `webapp.properties`. Can you potentially restore the old version, create a new file `webapps.docker` in the same folder and than just "rename" it to `webapps.properties`???

Thanks again!

Commit:
fixing Philip's comment + shortening the server starting time by using jetty-runner (https://mvnrepository.com/artifact/org.eclipse.jetty/jetty-runner)
Please note: use JettyStarter in RunInJettyPEPPOLAS2 from command line is hard because there is no dependency manager, you need to fix CLASSPATH manually or use IDE as Philip mentioned in README.md

izinin commented:
Hi Philip

Please see your comment fixed -- we do not need a separate path for `data` folder , Docker config can use original `webapp.properties` as it is -- i.e. i reverted path back to original `webapp.datapath = /var/www/peppol-as2/data` and do not need separate `webapps.docker` file. Sorry , originally it was my mistake.
Plus there is some improvement implemented in the Dockerfile definition that allows fast server startup time: instead of `mvn jetty:run` , that re-build the project , i use `jetty-runner` that can run `as2-peppol-server-*.war` directly

Sorry for my late response. I was busy with my daily  jobs.

Regards,
Igor

phax commented:
Hi Igor.
Your help is very much appreciated and we're definitively not in a hurry here :)
Thanks for taking the time to provide this 👍 
I'm also quite busy so my response times are also down the drain ;-)
// Philip
